modules/accredited_representative_portal/app/models/accredited_representative_portal/representative_user.rb

```diff
@@ -22,6 +22,7 @@ class RepresentativeUser < Common::RedisStore
     attribute :sign_in
     attribute :uuid
     alias_attribute :mhv_icn, :icn
+    attribute :verified
```

Comment:
It's not possible for a representative user to log in without being verified, so I don't think you need this attribute